packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1338,6 +1338,33 @@ module.exports.external = [
       return parseRange(obj);
     },
   },
+  {
+    name: "buildPresentedForm",
+    description: "Builds a presented form array",
+    func: function (b64String, component) {
+      const presentedForm = [];
+      if (b64String) {
+        presentedForm.push({
+          contentType: "text/plain",
+          data: b64String,
+        });
+      }
+      if (component) {
+        const components = Array.isArray(component) ? component : [component];
+        components.forEach(comp => {
```

Comment:
if you look at `getAllCdaSectionsByTemplateId` for ex. in this file, you will see that it does for loop to length -1 because "templateIds includes the full message at the end". This is a weird handlebars artifact where the last argument it passes in an object is like a meta object about the argument.  

Are we sure thats not the case here too? Can figure out yes or no by just printing components and checking to see there isnt a weird looking object at the end

Reply:
Checked, and there was no weird components printed this way